pcsx2/gui/MainMenuClicks.cpp

```diff
@@ -148,7 +148,7 @@ void MainEmuFrame::Menu_ResetAllSettings_Click(wxCommandEvent& event)
 		ScopedCoreThreadPopup suspender;
 		if (!Msgbox::OkCancel(pxsFmt(
 								  pxE(L"This command clears %s settings and allows you to re-run the First-Time Wizard.  You will need to manually restart %s after this operation.\n\nWARNING!!  Click OK to delete *ALL* settings for %s and force-close the app, losing any current emulation progress.  Are you absolutely sure?\n\n(note: settings for plugins are unaffected)"), WX_STR(pxGetAppName()), WX_STR(pxGetAppName()), WX_STR(pxGetAppName())),
```

Comment:
Shouldn't this line be also formatted?

Reply:
Seems it didn't trigger on pxE, but it does on macros.